lib/core/scheduler.rb

```diff
@@ -151,6 +155,51 @@ def configuration_triggered?(connector_settings)
       false
     end
 
+    def filtering_validation_triggered?(connector_settings)
+      return false unless connector_registered?(connector_settings.service_type)
+
+      filtering = connector_settings.filtering
+
+      unless filtering.present?
+        Utility::Logger.info("#{connector_settings.formatted} does not contain filtering to be validated.")
```

Comment:
Too often, you're right. Then I just change it to debug? WDYT?

Reply:
👍 sounds good. I'd also be fine with switching this message to just include the connector ID instead of the whole settings object. Either way is good.